File: BookStore.DataAccess/Models/Author.cs
namespace BookStore.DataAccess.Models
{
    public class Author : BaseEntity
    {
      public string FirstName { get; set; }
      public string SecondName { get; set; }
      public DateTime BirthDay { get; set; }
      public DateTime? DayOfDeath { get; set; }
      public virtual List<Ganre> Ganres { get; set; } = new List<Ganre>();
      public virtual List<Product> Products { get; set; } = new List<Product>();
    }
}

File: BookStore.Api/admin/src/shared/models/author/Author.ts
import { Ganre } from "../ganre/Ganre";

export interface Author {
    id: string,
    creationDate: string,
    isActive: boolean,
    isDeleted: boolean,
    firstName: string,
    secondName: string,
    birthDay: string,
    dayOfDeath?: string,
    ganres: Ganre[],
}